scripts/docker-get-logs.sh

```diff
@@ -3,8 +3,10 @@ set -euo pipefail
 
 STEP=${1:-""}
 
-mkdir -p docker-info/${STEP}
-cd docker-info/${STEP}
+DOCKR_INFO_DIR="docker-info/${STEP}"
+mkdir -p ${DOCKR_INFO_DIR}
+cp docker-compose.yml ${DOCKR_INFO_DIR}
```

Comment:
it copies the docker-compose-yml file to the docker-info folder to be archived on the CI